config/src/generator.rs

```diff
@@ -59,6 +68,17 @@ pub fn validator_swarm(
             network_keypairs.identity_keys.public().clone(),
         ));
 
+        // TODO(philiphayes): as a temporary hack, we'll just duplicate the
+        // validator info into the fullnode info until we can handle deserializing
+        // empty fullnode info.
+        discovery_infos.push(DiscoveryInfo {
+            account_address: network.peer_id,
+            validator_network_identity_pubkey: network_keypairs.identity_keys.public().clone(),
+            validator_network_address: network.advertised_address.clone(),
```

Comment:
I believe we should not be putting this into this genesis blob, instead placing a deterministic value.